src/panels/lovelace/common/graph/coordinates.ts

```diff
@@ -20,18 +20,18 @@ const calcPoints = (
     }
   });
   const rangeY = maxY - minY || minY * 0.1;
+  // add top margin to prevent cropping
+  maxY += rangeY * 0.1;
```

Comment:
Might as well add the bottom margin here as well. I had it like this initially and don't remember why I moved them in the if/else. The code has changed since then so I think it will be ok to add both margins here